agents/check_mk_agent.openwrt

```diff
@@ -528,11 +528,14 @@ if type chronyc > /dev/null 2>&1 ; then
     run_cached -s chrony 30 "waitmax 5 chronyc tracking || true"
 fi
 
-if type nvidia-settings >/dev/null && [ -S /tmp/.X11-unix/X0 ]
-then
-    echo '<<<nvidia>>>'
-    for var in GPUErrors GPUCoreTemp
-    do
+if inpath nvidia-smi; then
+    echo '<<<nvidia-smi>>>'
+    waitmax 2 nvidia-smi --query-gpu=temperature.gpu --format=csv,nounits,noheader | sed "s/^/GPUCoreTemp: /"
+fi
+
+if inpath nvidia-settings && [ -S /tmp/.X11-unix/X0 ]; then
```

Comment:
Oups, the inpath bash function is not defined in this script, so this won't work. I will fix that.

Why is there an NVIDIA check in this openwrt version? Does someone use openwrt on systems with NVIDIA GPUs?